oci/private/image.sh.tpl

```diff
@@ -12,6 +12,8 @@ readonly STORAGE_DIR="{{storage_dir}}"
 
 readonly STDERR=$(mktemp)
 
+trap "stop_registry ${STORAGE_DIR}" EXIT
```

Comment:
this overrides the existing trap on EXIT called `silent_on_success` can you just call the stop registry function there?